### ISSUE TITLE
Requirements extension does not cover full CCCEV data model

### ISSUE THREAD

**Issue by MusicMaker1337** [open-contracting/ocds-extensions#132]
The Requirements extension at this moment covers the following requirement-related CCCEV classes:
- `Criterion`
- `CriterionRequirement`
- `RequirementGroup`
- `RequirementResponse`

However, for some reason the extension does not cover `Evidence` and `DocumentReference` classes from the CCCEV model. It also does not cover `weight` and `criterionType` properties of the `Criterion` class.

I think it could be a good idea to make the extension cover the full CCCEV model

**Comment by jpmckinney** [comment]
Good point, @MusicMaker1337! Would you be interested in preparing a pull request with those changes?

**Comment by MusicMaker1337** [comment]
> Good point, @MusicMaker1337! Would you be interested in preparing a pull request with those changes?

Yes, I would definitely be interested. Could you please provide a link to the repo where the extension is located?

**Comment by jpmckinney** [comment]
Here it is: https://github.com/open-contracting-extensions/ocds_requirements_extension

**Comment by jpmckinney** [comment]
Noting that the CCCEV model has a 2.0 version with many changes: https://github.com/SEMICeu/CCCEV/blob/master/releases/2.00/Changelog.md
